apps/frontend/src/pages/people/[username].tsx

```diff
@@ -270,6 +262,23 @@ const UserProfilePage: React.FC = () => {
             setShowCommentModal(true);
           }
 
+          // If refreshPSI set to true, run refreshPSI and get most recent verified intersection
+          if (user && user.connections && user.connections[username]) {
+            const connection = user.connections[username];
+
+            if (connection.refreshPSI) {
```

Comment:
Here's the order of steps: 
1. Receive PSI message for connection X  
2. Refreshes PSI for connection X 
3. Set connection X's refreshPSI to true 
4. When we next navigate to X's page, pull the results by refreshing (we can't pull directly because there's no guarantee the PSI results will still exist in server memory). 

Currently (2) is a redundant refresh so I think I may remove it (but keep (3) so the second refresh still works).